notebooks/Training a Custom MONAI Bundle.ipynb

```diff
@@ -23,93 +23,86 @@
     "If you haven't already generated your key or if you're unsure about the process, follow our step-by-step for [Generating and Managing Your Credentials](./Generating%20and%20Managing%20Your%20Credentials.ipynb).\n"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='Setup'></a>\n",
+    "\n",
+    "### Setup"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import json\n",
     "import requests\n",
-    "\n",
+    "import json\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
     "# API Endpoint and Credentials\n",
-    "monai_cloud_api = \"<MONAI Cloud API URL>\"\n",
-    "api_url = f\"{monai_cloud_api}/api/v1\"\n",
+    "host_url = \"<MONAI Cloud API URL>\"\n",
     "ngc_api_key = \"<NGC API Key>\"\n",
-    "\n",
-    "# NGC UID \n",
-    "response = requests.get(f\"{api_url}/login/{ngc_api_key}\")\n",
-    "uid = response.json()[\"user_id\"]\n",
-    "token = response.json()[\"token\"]\n",
-    "\n",
-    "# Construct the URL and Headers\n",
-    "base_url = f\"{api_url}/user/{uid}\"\n",
-    "headers = {\"Authorization\": f\"Bearer {token}\"}"
+    "# Object storage info\n",
+    "container_url = \"<remote object storage address>\"\n",
+    "access_id = \"<user id>\"\n",
+    "access_secret = \"<storage secret>\"\n",
+    "# training parameters\n",
+    "epochs = 2"
```

Comment:
epochs seems weird here.  Would it make sense to move it to the "Training on DGX Cloud" code block?

Reply:
same comments as below.
Or maybe we can hard code the epochs into 2 in the training code block, and then training related variables don't need to override in the CI script. Hi @mingxin-zheng @zephyrie , what do you think?